### PR TITLE
[satisfactory] - feat: add satisfactory helm

### PR DESCRIPTION
**Pull request by Truxnell**
**Description of the change**

Add new chart for dedicated game server satisfactory

**Benefits**

New chart!!!

**Possible drawbacks**

Maintenance :(

**Applicable issues**

Nil

**Additional information**

Not so much - I haven't include a health check as its a little difficult to do with a UDP game server (without jumping through hoops - perhaps a bash script/2nd container to pgrep process or similar?)

**Checklist** 
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [X] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

File: charts/stable/satisfactory/values.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  # DISABLESEASONALEVENTS: false # disable the FICSMAS event (you miserable bastard)
+  # MAXPLAYERS: 4 # set the player limit for your server
+  # PGID: 1000 # set the group ID of the user the server will run as
+  # PUID: 1000 # set the user ID of the user the server will run as
+  # SERVERBEACONPORT: 15000 # set the game's beacon port
+  # SERVERGAMEPORT: 7777 # set the game's port
+  # SERVERIP: 0.0.0.0    # set the game's ip (usually not needed)
+  # SERVERQUERYPORT: 15777 # set the game's query port
+  # SKIPUPDATE: false # avoid updating the game on container start/restart
+  # STEAMBETA: false # set experimental game version
```
Comment (bjw-s) [inline charts/stable/satisfactory/values.yaml:37]: Any particular reason you did not go with the `# --` style comment formatting here?

File: charts/stable/satisfactory/values.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        port: 7777
+        protocol: UDP
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- See values.yaml
+persistence:
+  config:
+    enabled: false
+    mountPath: /config
+  # server download game files from steam to the below folder
```
Comment (bjw-s) [inline charts/stable/satisfactory/values.yaml:66]:
```suggestion
    mountPath: /config

    # server download game files from steam to the below folder
```

**Review by bjw-s — CHANGES_REQUESTED** [review]

File: charts/stable/satisfactory/values.yaml (hunk 1 above)
Comment (Truxnell) [reply charts/stable/satisfactory/values.yaml:37]: I simply overlooked it! Will fix.

Comment (Truxnell) [reply charts/stable/satisfactory/values.yaml]: Actually, im not sure how the template will go as the actual 'comment' is the second #, not the first (as im commenting out default ENV vars.  Commiting and well see?

**Review by Truxnell — COMMENTED** [review]
All fixed :)

**Review by bjw-s — APPROVED** [review]

**Pull request merged**